_platforms/it/spid.md

```diff
@@ -201,6 +213,7 @@ L'integrazione di SPID è consentita sia per i siti della Pubblica Amministrazio
 3. Usa un Identity Provider di test come [spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} oppure [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} per simulare il flusso di autenticazione e verificare che la tua implementazione sia corretta.
 4. Entra in contatto con gli altri sviluppatori in [Slack](https://slack.developers.italia.it/){:target="_blank"}.
 5. Segui la procedura di accreditamento [descritta nel sito di AGID](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. Se hai ulteriori domande o stai riscontrando problemi con la procedura di accreditamente consulta la pagina di servizio [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add).
```

Comment:
```suggestion
6. Se hai ulteriori domande o stai riscontrando problemi con la procedura di accreditamento consulta la pagina di servizio [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add).
```

Reply:
> We call it "accreditation procedure" in the previous point

I made a commit that normalize this to "onboarding", does it sound good to you?